Gems/PhysX/Code/Source/EditorShapeColliderComponent.cpp

```diff
@@ -440,25 +440,24 @@ namespace PhysX
 
         SetShapeConfig(ShapeType::Capsule, capsuleShapeConfig);
 
-        m_shapeConfigs.back()->m_scale = scale;
-        const float scalarScale = scale.GetMaxElement();
-        m_geometryCache.m_radius = scalarScale * capsuleShapeConfig.m_radius;
-        m_geometryCache.m_height = scalarScale * capsuleShapeConfig.m_height;
+        m_shapeConfigs.back()->m_scale = AZ::Vector3(scale);
+        m_geometryCache.m_radius = scale * capsuleShapeConfig.m_radius;
+        m_geometryCache.m_height = scale * capsuleShapeConfig.m_height;
     }
 
-    void EditorShapeColliderComponent::UpdateSphereConfig(const AZ::Vector3& scale)
+    void EditorShapeColliderComponent::UpdateSphereConfig(const float scale)
     {
         float radius = 0.0f;
         LmbrCentral::SphereShapeComponentRequestsBus::EventResult(radius, GetEntityId(),
             &LmbrCentral::SphereShapeComponentRequests::GetRadius);
 
         SetShapeConfig(ShapeType::Sphere, Physics::SphereShapeConfiguration(radius));
         
-        m_shapeConfigs.back()->m_scale = scale;
-        m_geometryCache.m_radius = scale.GetMaxElement() * radius;
+        m_shapeConfigs.back()->m_scale = AZ::Vector3(scale);
```

Comment:
question: Same here, is the `.back()` call always safe?

Reply:
See other comment.